employee-rostering-frontend/src/ui/pages/SkillsPage.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as React from 'react';
+import {DataTable, DataTableProps} from 'ui/components/DataTable';
+import {skillOperations} from 'store/skill';
+import Skill from 'domain/Skill';
+import { AppState } from 'store/types';
+import { IRow } from '@patternfly/react-table';
+import { connect } from 'react-redux';
+
+interface StateProps extends DataTableProps<Skill> {
+
+}
+
+function skillToTableRow(rowData: Skill) : IRow {
+  // @ts-ignore
+  return {
+    cells: [{title: <div>{rowData.name}</div>}],
+    isOpen: true,
+    props: {}
+  };
+} 
+
+const mapStateToProps = ({ skillList }: AppState): StateProps => ({
+  title: 'Skills',
+  columnTitles: ['Name'],
+  tableData: skillList.skillList,
+  rowDataToRow: skillToTableRow
+});
+
+export interface DispatchProps {
+  refreshSkillList: typeof skillOperations.refreshSkillList;
+}
+
+const mapDispatchToProps: DispatchProps = {
+  refreshSkillList: skillOperations.refreshSkillList
+};
+
+export type Props = StateProps & DispatchProps;
+
+class SkillsPage extends DataTable<Skill, Props> {
```

Comment:
Prefer composition over inheritance. You can have
```ts

class SkillsPage extends React.Component<Props> {
  render() {
    return <DataTable
      title={'Skills'}
      columnTitles={['Name']}
      tableData={this.props.skills}
      rowDataToRow={skillToTableRow}
    />
  }
}
```
which will make `SkillPage`'s props simpler because things like `title` and `columns` are constant. (You'll need to change `mapStateToProps` accordingly but it'll end up simpler because the only thing we need to pass from the `AppState` is the `skillList`.)